manifestservice/manifests/__init__.py

```diff
@@ -169,7 +169,7 @@ def _get_folder_name_from_token(user_info):
 
     According to the revproxy's helpers.js, it looks like the user_id is stored in a variable called "sub". Hm. 
     """
-    return "user-" + str(user_info["sub"])
+    return app.config["HOSTNAME"] + "-user-" + str(user_info["sub"])
```

Comment:
Two ideas:
* first - maybe introduce a new config variable, maybe 'prefix' or something like that, that we introduce for new commons.  I'm worried that just adding hostname across the board might break users that already have saved manifests
* second - instead of 'prefix-userId' do 'prefix/userId':
```
result =  "user-" + str(user_info["sub"])
if "PREFIX" in app.config:
  result =  app.config["PREFIX"] + "/user-" + str(user_info["sub"])
```

Reply:
ok done! redeployed to qa-dcp

`
qa-dcp@cdistest_dev_admin:~$ aws s3 ls manifest-qa-dcp-planx-pla-net
                           PRE a-prefix-value-zakir-set-in-config.json/
                           PRE qa-dcp.planx-pla.net-user-29/
                           PRE user-19/
                           PRE user-20/
                           PRE user-29/

qa-dcp@cdistest_dev_admin:~$ aws s3 ls manifest-qa-dcp-planx-pla-net/a-prefix-value-zakir-set-in-config.json/user-29/
2019-11-20 17:58:09        450 manifest-2019-11-20T17-58-08.900063.json
2019-11-20 17:58:30        750 manifest-2019-11-20T17-58-29.445463.json



`